stockfish/models.py

```diff
@@ -728,6 +719,208 @@ def will_move_be_a_capture(self, move_value: str) -> Capture:
         else:
             return Stockfish.Capture.NO_CAPTURE
 
+    def convert_human_notation_to_sf_notation(self, move: str) -> str:
+        """
+        Args:
+            move:
+                A string which is the notation for a move, in a format that's used
+                by humans. E.g., stuff like Nf3, bxe5, etc.
+        Returns:
+            A string representation the notation for this move, in the form that
+            SF uses. E.g., Nf3 might become g1f3.
+        """
+
+        move = move.replace(" ", "").replace("-", "")
+        while move[-1:] in ["+", "#"]:
```

Comment:
maybe include that in the regex in line 788 to be more restrictive of how many of such symbols may appear, then make sure to check if their presence (and possibly absence) is actually valid.

Reply:
@FalcoGer Thx for the feedback! I've implemented most of the suggestions and pushed. For the stuff with # + - ' ' characters, it's probably fine to not verify whether they should be there, since they don't give any extra info on what move is being described (from the perspective of the function anyway).